### ISSUE TITLE
fetching certificate - lacking cache

### ISSUE THREAD

**Issue by nikooo777** [lbryio/lbry#683]
## The Issue
When calling claim_list_mine, and perhaps other methods too, if there are several claims sharing a certificate, the daemon will try to fetch such certificate for as many times as it's been used.
This takes a HUGE amount of time for big channels.
### Steps to reproduce
1. publish 2000 videos with one certificate
2. call claim_list mine
3. observe the daemon

### Expected behaviour
The daemon should cache the certificate for as long as it might be needed, ie. for the duration of the call and thus the certificate would only be fetched once

### Actual behaviour

The daemon fetches the certificate n times and it takes forever
```
fetching certificate to check claim signature
fetching certificate to check claim signature
fetching certificate to check claim signature
fetching certificate to check claim signature
fetching certificate to check claim signature
fetching certificate to check claim signature
...
```

## System Configuration

```
{
  "id": null,
  "jsonrpc": "2.0",
  "result": {
    "build": "rc",
    "ip": "34.207.2****",
    "lbrynet_version": "0.12.0rc1",
    "lbryschema_version": "0.0.7",
    "lbryum_version": "2.8.0",
    "os_release": "4.4.0-78-generic",
    "os_system": "Linux",
    "platform": "Linux-4.4.0-78-generic-x86_64-with-Ubuntu-16.04-xenial",
    "processor": "x86_64",
    "python_version": "2.7.12"
  }
}
```

**Comment by lyoshenka** [comment]
@jackrobison how long would it take to add caching here too

**Comment by jackrobison** [comment]
I'll check it out - it shouldn't require any network requests.

@nikooo777 is the certificate claim in that wallet too, or just the certificate key?

**Comment by lyoshenka** [comment]
@jackrobison is this fixed? or will this be fixed by https://github.com/lbryio/lbryum/issues/109 ?

**Comment by alyssaoc** [comment]
closed by #109
